### ISSUE TITLE
Missing theater mode button on control

### ISSUE THREAD

**Issue by fireattack** [EchoDev/TwitchHTML5#12]
Title basically said all.

But thanks for this great script, it totally saved my experience on my old notebook!

**Comment by onetimesignup** [comment]
It may have been removed temporairly due to all the new stuff

for now you can use the alt + T key combination to de/activate the theatre mode,
also alt + R shows/hides right panel/chat, and alt + L left panel/menu =)

**Comment by EchoDev** [comment]
Have you updated to the latest version?

Also what OS and Browser are you using?

**Comment by onetimesignup** [comment]
but you used to have a button below the message box in the chat that worked

**Comment by EchoDev** [comment]
Yeah the button in the chat box works but I can also make a button pop up next to the quality and fullscreen button (The one that shows up in the Flash version). That one doesn't work.

**Comment by onetimesignup** [comment]
another idea would be to put it next to following share etc where twitch used to have it, though I wasnt really a big fan of this place though cause the buttons in that area tend to load in slowly so you somtimes end up pressing wrong and hit like follow or subscribe or the gear.

i like the chat box area but maybe replace it with the new small widescreen icon instead so it isnt as much "in your face".

**Comment by EchoDev** [comment]
Putting it in the share, follow area would make the button disappear in theater mode so I would need to create a new overlay button when it goes into theater mode. I tried replacing the button with an image but it sometimes bugged out but the text button always seems to work.

**Comment by onetimesignup** [comment]
was the image loaded in externally ? if it was maybe storing/loading the image as data uri would solve it

**Comment by Boru91** [comment]
works great :) Is a chance to make script for hitbox?

**Comment by EchoDev** [comment]
@onetimesignup I think it has to do with loading times, sometimes it just borks.

@Boru91 I never use hitbox so I wouldn't know, you are free to change the script to make it work on hitbox.

**Comment by onetimesignup** [comment]
yea i figured as much that's why I figured you could try adding/running the svg or png internal in the script, like it would be 1-2 kb of rawdata which is permastored, I just couldn't find the svg/png to experiment with it, with css you use: `url("data:image/png;base64,raw-data-goes-here")`

hitbox just adds &html5=true to their urls but then this wouldn't be the "twitch html5" player script.
the fix would be to update the regexp and then create conditions for each site, the html5 player on hitbox is a lot easier to implement as you don't have to do load it in with iframes, it's really just updating the url.

**Comment by onetimesignup** [comment]
back to the original topic

how about hooking onto the new twitch whisper icon and add theatre button next to it ?

**Comment by EchoDev** [comment]
People would be confused when 2 of the same icon's pop up. I tried moving the onstream control button to the chat but it didn't load at all (Due to the iframe limitation I guess.)

If I can find a proper Unicode symbol that fits the Theater mode I guess I can change it to that. Anything that increases load time by too much make the script fail too often.
